frontend/packages/kubevirt-plugin/src/components/vm-details.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { VmDetails, getName, getNamespace, getResource, DASHES } from 'kubevirt-web-ui-components';
+
+import {
+  ResourceLink,
+  navFactory,
+  LoadingInline,
+  NodeLink,
+  Firehose,
+  StatusBox,
+} from '@console/internal/components/utils';
+
+import { PodKind, k8sPatch, k8sGet, K8sResourceKindReference } from '@console/internal/module/k8s';
+import { PodModel, ServiceModel, NamespaceModel } from '@console/internal/models';
+import { breadcrumbsForOwnerRefs } from '@console/internal/components/utils/breadcrumbs';
+import { DetailsPage } from '@console/internal/components/factory';
+
+import { VmKind, VmiKind } from '../types';
+import { VirtualMachineInstanceModel, VirtualMachineInstanceMigrationModel } from '../models';
+
+// import { VmEvents } from './vm-events';
+// import VmConsolesConnected from '../vmconsoles';
+// import { Nic } from '../nic';
+// import { Disk } from '../disk';
+// import { menuActions } from './menu-actions';
+
+export const VmDetailsFirehose = ({ obj: vm }: { obj: VmKind }) => {
+  const { name, namespace } = vm.metadata;
+
+  const vmiRes = getResource(VirtualMachineInstanceModel, {
+    name,
+    namespace,
+    isList: false,
+    props: 'vmi',
+  });
+  vmiRes.ignoreIfMissing = true;
```

Comment:
`optional`

Add this to declaration. No reason this needs to be on a separate line.